packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -693,7 +709,7 @@ contract OPContractsManager is ISemver {
                                 gameConfig.disputeMaxClockDuration,
                                 gameConfig.vm,
                                 outputs[i].delayedWETH,
-                                getAnchorStateRegistry(fdg),
+                                getAnchorStateRegistry(pdg),
```

Comment:
The ASR for the PDG and FDG should be the same, so this change doesn't matter in practice, but curious why you didn't keep this as `getAnchorStateRegistry(fdg)` which feels more semantically clear?

Reply:
this changed with the varname change above, so the semantic diff is actually smaller. 

But now that we're getting `fdg` just above, I think that makes sense.